backend/src/main/java/com/zeros/backend/models/Restaurant.java

```diff
@@ -1,15 +1,20 @@
-package com.zeros.backend;
+package com.zeros.backend.models;
 
 import java.util.List;
 
-public class Restaurant {
+public class Restaurant extends Entity {
 
     private String name;
     private Location location;
     private float rating;
     private List<String> photosReference;
 
-    Restaurant(String name, Location location, float rating, List<String> photos) {
+    public Restaurant(String name, Location location, float rating, List<String> photos) {
+        this(null, name, location, rating, photos);
```

Comment:
because currently restaurants returned by google maps don't have an id, later we are going to refacto that part to make restaurants returned have also an id, they must exist on our database

Reply:
it's more clean to generate ID for restaurants returned by google map

Reply:
yes, google's restaurant info should be wrapped into a DTO containing our own id which is entity's one. The constructor which doesn't contain id will be removed later

Reply:
Good